.travis.yml

```diff
@@ -70,18 +70,38 @@ matrix:
           env:
               - REGULAR_TEST=false
               - PUBLISH_DOC=true
+        - name: WASM online demo
+          language: rust
+          rust: nightly
+          cache: cargo
+          script:
+              - cd wasm
+              - bash release.sh
+          env:
+              - REGULAR_TEST=false
+              - PUBLISH_DEMO=true
     allow_failures:
         - rust: nightly
           env: REGULAR_TEST=true
 
 deploy:
-  provider: pages
-  repo: RustPython/website
-  target-branch: master
-  local-dir: target/doc
-  skip-cleanup: true
-  github-token: $WEBSITE_GITHUB_TOKEN  # Set in the settings page of your repository, as a secure variable
-  keep-history: true
-  on:
-    branch: master
-    condition: $PUBLISH_DOC = true
+  - provider: pages
+    repo: RustPython/website
+    target-branch: master
+    local-dir: target/doc
+    skip-cleanup: true
+    github-token: $WEBSITE_GITHUB_TOKEN  # Set in the settings page of your repository, as a secure variable
+    keep-history: true
+    on:
+      branch: master
```

Comment:
I suggest still to create a seperate branch next to master which we can use to release less frequent to the demo site. Maybe we could introduce a `release` branch? We could then manually merge the master branch once or twice a month to update the latest release branch status. This way we would save cpu cycles on travis services.

Reply:
Makes sense. Let me change this. After merge I'll manually push a branch.